src/Script.cpp

```diff
@@ -49,7 +49,8 @@ Script::Script(const std::string& name, const std::string& mainFilePath, const s
 	{
```

Comment:
The reason this document has SO many changes is because I pressed "format file" in MSVC and it changed the spacing / tabulation of many functions

The only real changes are lines 398 to 496